tests/__snapshots__/no-duplicated-vue-import.spec.ts.snap

```diff
@@ -0,0 +1,31 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`no-duplicated-vue-import should remove duplicated global vue import 1`] = `
+"\\"use strict\\";
+Object.defineProperty(exports, \\"__esModule\\", { value: true });
+var vue1 = require(\\"vue\\");
+var vue2 = require('vue');
+var vue3 = require(\\"vue\\");
+exports.default = vue_2.defineComponent({
```

Comment:
Hmm actually this vue_2 should be vue_1 (which is global import

Reply:
I am thinking about removing all imports and reference all of usage to global Vue import vue_1

Reply:
Seems like ideally we should only have one import.

Reply:
ok I will try to look in that way